bgpd/bgpd.c

```diff
@@ -6785,19 +6786,19 @@ int peer_clear_soft(struct peer *peer, afi_t afi, safi_t safi,
 					       PEER_STATUS_ORF_PREFIX_SEND))
 					bgp_route_refresh_send(
```

Comment:
Sending of route refresh capability for respective AFI-SAFI should only be done if that AFI-SAFI is negotiated with this peer.